### DISCUSSION TITLE
Is there a way to restart the timing session to the start of the race?

### DISCUSSION THREAD

**Discussion by bwbang** [slowlydev/f1-dash#143] [Q&A] [answered]
As my title asks.
I can’t watch the GP live because of my time zone but I’d like use to F1-Dash to follow along when I watch the re-broadcast on TV. 
Where can I restart the timing?
And also can you not display the final results to prevent spoilers?

**Comment by SpatzlHD** [comment, marked as answer]
Currently there is no way to replay the timing data of a race after the race has finished but I am currently working on a feature that would allow for the reply of races from the past. There is an open Issue for it so follow that if you want to follow the progress #37
